lib/index.js

```diff
@@ -137,6 +150,10 @@ export function genderRoll(text, joined = true, pickerFunc = _.sample) {
     return null;
   }
   const slots = applyModifiers(_.flatten(rolls));
-  const selections = slots.map(pickerFunc);
+  var selections = slots.map(pickerFunc);
+  if (selections[1] == 'Color') {
+    selections.splice(1, 1, pickerFunc(COLORS));
```

Comment:
I think I would prefer to see this implemented by allowing modifiers to provide callbacks which will mutate the response.

Thus `YES_COLORS` would have a callback function which contains this `if` block. That will also make it much easier to write a test, since you can simply pass in selections that do contain "Color" and selections that don't, and make sure the results are appropriate.

BTW, I would appreciate a test being written for this. You can run the tests with `npm test`. I believe you can also use `gulp watch` to run the tests whenever something changes in the source or test files.